projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -131,6 +138,7 @@ public function get_track_data( $guid ) {
 			set_transient( $transient_key, $track_data, HOUR_IN_SECONDS );
 		}
 
+		remove_action( 'wp_feed_options', array( __CLASS__, 'reset_simplepie_cache' ) );
```

Comment:
Should we only do this when `$force_refresh` is set to true?

Reply:
Addressed in https://github.com/Automattic/jetpack/pull/18573/commits/6f84c4d0971772d6f34a59137a7410a814d8d191